Update hardware-transcoding.md (#9094)
immich-app/immich: docs/docs/features/hardware-transcoding.md

Once this is done, you can continue to step 3 of "Basic Setup".

> #### Immich Docker for Intel QuickSync - Unraid Setup 

Reviewer: ```suggestion
#### Intel QuickSync
```

Reviewer 2: It should just be `QSV` to be consistent with the other headers.

1. Unraid > Docker > (Stop) Immich container > Edit

> 2. Scroll down and select "Add another path..etc..device"

Reviewer 2: ```suggestion
2. Scroll down and select `Add another Path, Port, Variable, Label or Device`
```

> 3. In the popup drop down menu select Device
> 
>     Name - dev/dri (you are free to put whatever you want) 
>     Value - /dev/dri/
> 

Reviewer 2: ```suggestion
3. In the drop-down menu, select `Device` and an entry with any name and the value `/dev/dri`.
```

> 4. In the Immich web interface : Administration > Settings (left side panel) > Video Transcoding > Hardware Acceleration
> 5.  Select - Acceleration Api (QuickSync) then in the "PREFERRED HARDWARE DEVICE FOR TRANSCODING" field you can leave that auto
> 
> 

Reviewer 2: ```suggestion
4. Continue to step 4 of "Basic Setup".
```
